src/config.c

```diff
@@ -610,7 +611,9 @@ char *parse_config(options_t *opt, const char *config_file, bool *shadow_enable,
 
 	    .track_leader = false,
 
-	    .rounded_corners_blacklist = NULL
+	    .rounded_corners_blacklist = NULL,
+
```

Comment:
Nitpick, but can you remove this newline betwen `.rounded_corners_blacklist` and `.animation_blacklist`?